scapy/contrib/cansocket_python_can.py

```diff
@@ -141,9 +142,13 @@ class PythonCANSocket(SuperSocket, SelectableObject):
     nonblocking_socket = True
 
     def __init__(self, **kwargs):
+        self.args = copy.deepcopy(kwargs)
```

Comment:
I think this line breaks the CCP unit tests. Maybe you can check the basecls parameter. This leads probably to the error. I assume the basecls isn't set correctly in the test cases anymore.

Reply:
Interesting, I will try another method to see if anything changes.

Reply:
After a lot of testing I found out, that the breaking part happens not here, but in isotp.uts. the python builtin exec() that I use for reconstructing the ISOTPSockets breaks stuff in the interpreter but only in python2 and pypy (it is implementation dependent). I tried to somehow get it work in py2 and pypy but couldn't find a way, so I just disabled the test there.